.travis.yml

```diff
@@ -8,7 +8,7 @@ os:
   - linux
 #  - osx
 julia:
-  - 1.4
+  - 1.5
```

Comment:
I guess we might want to change (back?) to
```suggestion
  - 1
```
such that always the stable Julia version is tested automatically.

Reply:
We can have a separate testing for this, for this PR, I am aiming to see what issues we have in Julia 1.5 tests. See the travis tests, they are all red

Reply:
Travis will use Julia 1.5 when you use `- 1`.

Reply:
Ohh Travis follows semver, then we should do it, you are right